libs/shared/utils/src/maths.ts

```diff
@@ -0,0 +1,6 @@
+export const subtractPercentage = (value = 0, percent = 0) => {
```

Comment:
I am glaring at this function right now. (lol)

Reply:
yea, not very proud of it... I'd still want to go for [dnum](https://github.com/bpierre/dnum) or wait until it moves to [ECMA](https://github.com/tc39/proposal-decimal)